hack/bundle/render_templates.sh

```diff
@@ -16,54 +16,59 @@ ANNOTATIONS_FILE_NAME="annotations.yaml"
 
 update_csv_manifest()
 {
-  CSV_FILE="${MANIFESTS_DIR}/${CSV_FILE_NAME}"
-  if [[ ! -f ${CSV_FILE} ]]; then
-    echo "[$(date)] -- ERROR -- operator csv file \"${CSV_FILE}\" does not exist"
-    exit 1
-  fi
-
-  ## replace cert-manager operand related images
-  sed -i "s#quay.io/jetstack/cert-manager-webhook.*#${CERT_MANAGER_WEBHOOK_IMAGE}#g" ${CSV_FILE}
-  sed -i "s#quay.io/jetstack/cert-manager-controller.*#${CERT_MANAGER_CONTROLLER_IMAGE}#g" ${CSV_FILE}
-  sed -i "s#quay.io/jetstack/cert-manager-cainjector.*#${CERT_MANAGER_CA_INJECTOR_IMAGE}#g" ${CSV_FILE}
-  sed -i "s#quay.io/jetstack/cert-manager-acmesolver.*#${CERT_MANAGER_ACMESOLVER_IMAGE}#g" ${CSV_FILE}
-
-  ## replace kube-rbac-proxy image
-  sed -i "s#gcr.io/kubebuilder/kube-rbac-proxy.*#${KUBE_RBAC_PROXY_IMAGE}#g" ${CSV_FILE}
-
-  ## replace cert-manager-operator image
-  sed -i "s#openshift.io/cert-manager-operator.*#${CERT_MANAGER_OPERATOR_IMAGE}#g" ${CSV_FILE}
-
-  ## add annotations
-  yq e -i ".metadata.annotations.createdAt=\"$(date -u +'%Y-%m-%dT%H:%M:%S')\"" "${CSV_FILE}"
-
-  ## remove non-required fields
-  yq -i 'del(.spec.relatedImages)' "${CSV_FILE}"
```

Comment:
Need to add back the correct value of `.spec.relatedImages` towards the end of this block;
it's supposed to be a part of the bundle CSV based upon the other 3 images built.

Reply:
Ahh, missed it. Cool